templates/login.html

```diff
@@ -0,0 +1,34 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>Title</title>
+</head>
+<body>
+    {% block content %}
+    <h1>Login</h1>
```

Comment:
После этого флешки можно добавить тут.
```suggestion
    {% block content %}
        <h1>Login</h1>
    {% with messages = get_flashed_messages() %}
      {% if messages %}
      <ul>
          {% for message in messages %}
          <li>{{ message }}</li>
          {% endfor %}
      </ul>
      {% endif %}
      {% endwith %}
```